x-pack/plugins/enterprise_search/public/applications/analytics/components/analytics_collection_view/analytics_collection_integrate/analytics_collection_integrate_searchui.tsx

```diff
@@ -124,6 +124,16 @@ plugins: [
 ...
 }`}
           </EuiCodeBlock>
+          <EuiSpacer size="m" />
+          <p>
+            {i18n.translate(
+              'xpack.enterpriseSearch.analytics.collections.collectionsView.integrateTab.searchui.stepTwo.moreInfoDescription',
+              {
+                defaultMessage:
+                  'See the Javascript Embed and Javascript Client tabs for more information on initializing the tracker and firing events.',
```

Comment:
Here I refer to the use of `trackEvent` and `trackPageView` functions, which depends on the chosen integration solution (JS Embed vs Client).